src/composables/deeplink-api.ts

```diff
@@ -0,0 +1,33 @@
+import { ref } from '@vue/composition-api';
```

Comment:
This is a general comment for the PR: what about other components that is using the deepling mixin?